layers/bromley.map

```diff
@@ -134,4 +134,21 @@ MAP
     END
   END #layer
 
+  LAYER
+    NAME "Parks_Open_Spaces"
+    METADATA
+      "wfs_title"         "Parks & open spaces" ##REQUIRED
+      "wfs_srs"           "EPSG:3857" ## REQUIRED
+      "gml_include_items" "SITENAME,site_code" ## Optional (serves all attributes for layer)
```

Comment:
I realise this isn't used yet, but perhaps worth adding a line here `"gml_SITENAME_alias" "site_name"` (or similar) so that at the client side these two data bits are sent as site_name and site_code.